games-board/stockfish/stockfish-sf_8.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="A chess engine claimed to be strongest in the world"
+DESCRIPTION="Stockfish is a free UCI chess engine derived from Glaurung 2.1. It \
+is not a complete chess program and requires some UCI-compatible GUI (e.g. \
+XBoard with PolyGlot, eboard, Arena, Sigma Chess, Shredder, ChessPartner or \
+Fritz) in order to be used comfortably. Read the documentation for your GUI of \
+choice for information about how to use Stockfish with it."
+HOMEPAGE="https://www.stockfishchess.org/"
+COPYRIGHT="2004-2008 Tord Romstad (Glaurung author)
+	2008-2017 Marco Costalba, Joona Kiiski, Gary Linscott, Tord Romstad"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/official-stockfish/Stockfish/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="e2b77d801d8837c22c2391b8cae08622c1434a2bb5947ddb08b081e1ad35c76c"
+SOURCE_DIR="Stockfish-sf_8"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64 ?arm ?ppc"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
```

Comment:
If you tried to build on x86_gcc2 using the primary arch, that is, building stockfish (and not stockfish_x86) and noticed that the source is not compatible with gcc2, then you should keep SECONDARY_ARCHITECTURES. Your current definition for it looks OK, btw.

If, on the other side, it builds fine on x86_gcc2 primary arch, then there is no need to keep SECONDARY_ARCHITECTURES and you should, in that case, remove it.